integration/env/common.go

```diff
@@ -13,6 +13,9 @@ const (
 	// EnvVarCircleSHA is the process environment variable representing the
 	// CIRCLE_SHA1 env var.
 	EnvVarCircleSHA = "CIRCLE_SHA1"
+	// EnvVarE2EKubeconfig is the process environment variable representing the
+	// E2E_KUBECONFIG env var.
+	EnvVarE2EKubeconfig = "E2E_KUBECONFIG"
```

Comment:
We were using the `e2esetup` lib just for this env var. Moving it locally to simplify the dependencies.